### PR TITLE
Corrected all the remaining title-case-style capitalization

### PR DESCRIPTION
**Pull request by radinamatic**
### Summary

While translating user docs in Spanish I've found several occurrences of Kolibri user strings with a title case used instead of the sentence-style capitalization we are supposed to be using according to [Material guidelines](https://material.io/guidelines/style/writing.html#writing-capitalization-punctuation).

Here are all squashed, but it's quite a hammer, so let's discuss it if necessary.

Some examples:

selection_174

selection_178

### Contributor Checklist

- [x] PR has the correct target branch and milestone
- [x] PR has 'needs review' or 'work-in-progress' label
- [x] Contributor has fully tested the PR manually
- [x] Screenshots of any front-end changes are in the PR description
- [x] If PR is ready for review, a reviewer has been added. (Don't use 'Assignees')

### Reviewer Checklist

- [ ] Automated test coverage is satisfactory
- [ ] Reviewer has fully tested the PR manually
- [ ] PR has been tested for [accessibility regressions](http://kolibri-dev.readthedocs.io/en/develop/manual_testing.html#accessibility-a11y-testing)
- [ ] External dependencies files were updated (`yarn` and `pip`)
- [ ] Documentation is updated
- [ ] Link to diff of internal dependency change is included
- [ ] CHANGELOG.rst is updated for high-level changes
- [ ] Contributor is in AUTHORS.rst

cc @indirectlylit @christianmemije @jonboiser @jtamiace @khangmach

### CODE REVIEW

File: kolibri/plugins/coach/assets/src/views/exams-page/index.vue [hunk 1]
```diff
@@ -118,11 +118,11 @@
     name: 'coachExamsPage',
     $trs: {
       exams: 'Exams',
-      classExams: '{className} Exams',
+      classExams: '{className} exams',
```
Comment (indirectlylit) [inline kolibri/plugins/coach/assets/src/views/exams-page/index.vue]: while we're here, maybe we can change this to:

```
classExams: "Exams in '{className}'",
```

Comment (radinamatic) [reply kolibri/plugins/coach/assets/src/views/exams-page/index.vue]: Makes total sense, will change asap!

**Comment by jonboiser** [comment]
@radinamatic The merge conflicts are due to the recent splitting of the two management pages. I _hope_ a simple merge or rebase will take care of it...

**Comment by indirectlylit** [comment]
looks great

**Pull request merged**